docs/v3.x/plugins/email.md

```diff
@@ -97,10 +97,11 @@ The string after the last `@` represents your desired [semver](https://docs.npmj
 ### Configure your provider
 
 After installing your provider you will need to add some settings in `config/plugins.js`.
-If this file doesn't exists, you'll need to create it.
 
-> ⚠️ Do note that filename has to come with correct spelling, plugin with 's' (plural). -> `plugins.js`
-> Check the README of each provider to know what configuration settings the provider needs.
+::: warning
+Do note that filename has to come with correct spelling, plugin with 's' (plural). -> `plugins.js`
```

Comment:
I would suggest: "Filenames must come with the correct spelling, and plugins must be in plural form (e.g. `plugins.js`)."

Reply:
I only changed the quote into a `::: warning` :wink:, but am for sure can change it. 

That filenames in general must come with correct spelling is kinda obvious, this warning or more so a tip that explicitly is about `plugins.js` because there where a couple of people that created the file without an 's'. (related: #7769) 

Maybe this is better:
```markdown
After installing your provider you will need to add some settings in `config/plugins.js`.
Check the README of each provider to know what configuration settings the provider needs.

::: tip
Make sure you have the correct spelling of the configuration filename, it is written in plural (with a trailing 's'): `plugins.js`.
:::
```

Reply:
Alright! Your suggestion looks good to me 👍